pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -236,7 +239,11 @@ func reportMetrics(pa *pav1alpha1.PodAutoscaler, pc podCounts) error {
 //    | -1   | >= min | >0    | active     | active     |
 func computeActiveCondition(ctx context.Context, pa *pav1alpha1.PodAutoscaler, pc podCounts) {
 	minReady := activeThreshold(ctx, pa)
-	if pc.ready >= minReady {
+	// In pre-0.17 we could have scaled down normally without ever setting ScaleTargetInitialized.
+	// In this case we'll be in the NoTraffic/inactive state.
+	// TODO(taragu): remove after 0.19
```

Comment:
Can we open a tracking issue?  Should it be in 0.19?  I don't understand after.

Reply:
It should be in 18 for sure.  We can keep it in 19, if we want to, or we can eliminate in 0.20.
I'll file an issue.

Reply:
https://github.com/knative/serving/issues/9114